backend/module-campaigns/src/Podkrepibg.Campaigns.Application/Podkrepibg.Campaigns.Application.csproj

```diff
@@ -16,12 +16,8 @@
     <PackageReference Include="MediatR.Extensions.Microsoft.DependencyInjection" Version="9.0.0" />
     <PackageReference Include="Microsoft.EntityFrameworkCore" Version="5.0.5" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="5.0.0" />
-    <PackageReference Include="Grpc.AspNetCore" Version="2.37.0" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <Protobuf Include="..\..\..\protos\campaign.proto" GrpcServices="Server" ProtoRoot="..\..\..\protos\" />
-    <Protobuf Include="..\..\..\protos\common\nomenclatures.proto" GrpcServices="Server" ProtoRoot="..\..\..\protos\" />
+    <PackageReference Include="Grpc.AspNetCore" Version="2.38.0" />
```

Comment:
maybe we can remove this nuget from here as it is no longer needed? We can put it in the main project (Podkrepibg.Campaigns). If we do so, we have to add a reference to `Microsoft.Extensions.Configuration.Abstractions` package here.